source/seismology/seismic-wave.rst

```diff
@@ -1,2 +1,211 @@
 地震波
 ======
+
+:本节贡献者: |姚家园|\（作者）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-05-30
+:预计阅读时间: 20 分钟
+
+----
+
+地震波（seismic wave）是在地球内部传播的机械波，由地震、火山喷发、山体滑坡或人为爆炸等震源激发。
+地震波分为体波（body wave）和面波（surface wave）。体波是指在地球内部传播的地震波，
+而面波指在地球表面附近传播的地震波。本节将介绍最基础的地震波理论。
+
+体波
+-----
+
+P 波与 S 波
+^^^^^^^^^^^^
+
+体波分为 P 波（\ **p**\ rimary wave 或 P-wave）和 S 波（\ **s**\ econdary wave 或 S-wave）。
+P 波比其他地震波传播速度快，最先传播到达台站；S 波的传播速度比 P 波慢，在 P 波后传播到台站。
+上述 P 波和 S 波的命名源由主要基于他们到达台站的时间。
+从物理属性上来说，P 波是压缩波（compressional wave），这是因为 P 波在地下介质中传播时，
+会造成介质的交替压缩和膨胀，介质的体积发生变化（如下图示）；S 波是剪切波（\ **s**\ hear wave），
+这是因为 S 波在地下介质中传播时，会造成介质的剪切变形，但介质的体积没有变化（如下图示）。
+
+P 波之所以会造成介质的交替压缩和膨胀，是由于介质的运动方向与波传播的方向相同或相反。
+因此，P 波是一种纵波（longitudinal wave），即介质的运动方向与波传播的方向相同或相反的波。
+S 波之所以会造成介质的剪切变形，是由于介质的运动方向与波传播的方向垂直。
+因此，S 波是一种横波（transverse wave），即介质的运动方向与波传播的方向垂直的波。
+
+.. figure:: body-wave-propagation.jpg
+   :alt: 体波的介质运动方向以及波传播方向
+   :width: 80%
+   :align: center
+
+   体波的介质运动方向以及波传播方向。引自《\ `Introduction to Seismology`_\ 》（第三版）图 3.2。
+   
+.. _Introduction to Seismology: https://www.cambridge.org/us/academic/subjects/earth-and-environmental-science/solid-earth-geophysics/introduction-seismology-3rd-edition?format=HB&isbn=9781316635742
+
```

Comment:
Personally, I prefer to move the link definition to the end of this file, and also add the link definitions for "An Introduction to Seismology, Earthquakes, and Earth Structure" and "An Introduction to the Theory of Seismology" to make the figure caption shorter.

Reply:
Okay

Reply:
Done in 41d56d3.